distributedlog-core/src/main/java/org/apache/distributedlog/impl/BKNamespaceDriver.java

```diff
@@ -250,11 +252,22 @@ private synchronized BKDLConfig getBkdlConfig() {
         return bkdlConfig;
     }
 
+    static EventLoopGroup getDefaultEventLoopGroup(int numThreads) {
+        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("DL-io-%s").build();
+        if (SystemUtils.IS_OS_LINUX) {
+            try {
+                return new EpollEventLoopGroup(numThreads, threadFactory);
+            } catch (Throwable t) {
+                LOG.warn("Could not use Netty Epoll event loop for bookie server: {}", t.getMessage());
```

Comment:
Any reason to not print the stack?

LOG.warn("Could not use Netty Epoll event loop for bookie server: ", t);